week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -24,27 +23,27 @@ function greaterThan10() {
 
 function sortArray() {
   let letters = ["a", "n", "c", "e", "z", "f"];
-  let sortedLetters;
+  let sortedLetters = letters.sort();
 
   return sortedLetters;
 }
+console.log(sortArray());
```

Comment:
It's best practice to remove `console.log` entries after you complete testing so that other developers don't get confused